entries/entry04.md

```diff
@@ -1,7 +1,29 @@
-# Entry 4
-##### X/X/XX
+# Entry 4: Learning new components
+##### 2/10/20
 
-Text
+I was putting up the monthly school calendar on the Student Government bulletin board a few weeks ago when I thought, students get most of their information from their phones. Why not have all school events on one place on a student's phone? Like a Tele Kids App!<br>
+
+My partner, Karla and I wrote down all of our ideas for the app but then realized we would need a MVP (minimum valuable product) before reaching to our highest expectations of the app. We came up with a short list that includes the following:<br>
+![MVP](MVP.png)<br>
```

Comment:
It might have looked a bit nicer to just copy/paste and use markdown syntax for your list.